### DISCUSSION TITLE
How to make fine-tuning for QA using Arabert

### DISCUSSION THREAD

**Discussion by EssamKayal96** [aub-mind/arabert#103] [Q&A] [answered]
Hi ^_^ i'am new here i have a QA problem and i want to fine-tuning with arabert , any help or mention

**Comment by WissamAntoun** [comment, marked as answer]
If you have the data formated as in SQuAD, you can use the QA colab in the examples folder `https://github.com/aub-mind/arabert/tree/master/examples`
